### PR TITLE
Add deepface, retina-face

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
I'd remove `pip check` from `deepface` as the conda-forge package doesn't follow the non-standard versioning of upstream:
```
import: 'deepface'
+ pip check
deepface 0.0.75 has requirement opencv-python>=4.5.5.64, but you have opencv-python 4.5.5.
```

**Comment by BastianZim** [comment]
Hmm yeah, you’re right that's an easy fix. But why did I see a different error this morning…? 😅

**Comment by dhirschfeld** [comment]
Windows is still causing issues. Having a look to see if it's not too difficult to track down...

**Comment by BastianZim** [comment]
Ahh, that’s where the error was. I thought I was going insane. 😄
I think some deps aren‘t available — let me check.

**Comment by BastianZim** [comment]
Actually, I think that’s the Tensorflow dep. I'll add a noarch patch and then we should be fine.

**Comment by BastianZim** [comment]
@conda-forge/help-python This is ready for review, thanks!

File: recipes/deepface/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+about:
+  home: https://github.com/serengil/deepface
+  summary: A Lightweight Face Recognition and Facial Attribute Analysis Framework (Age, Gender, Emotion, Race) for Python
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - serengil
```
Comment (BastianZim) [inline recipes/deepface/meta.yaml:55]: Agreed in https://github.com/serengil/deepface/issues/486#issuecomment-1137767097

**Comment by BastianZim** [comment]
@conda-forge/staged-recipes Gentle ping

**Pull request merged**